Neos.Flow/Classes/Command/PackageCommandController.php

```diff
@@ -103,46 +103,26 @@ public function createCommand(string $packageKey, string $packageType = PackageI
     /**
      * List available packages
      *
-     * Lists all locally available packages. Displays the package key, version and
-     * package title.
+     * Lists all locally available packages. Displays the package key, installed version and package title.
      *
-     * @param boolean $loadingOrder The returned packages are ordered by their loading order.
      * @return void The list of packages
      */
-    public function listCommand(bool $loadingOrder = false)
+    public function listCommand(): void
     {
-        $availablePackages = [];
-        $frozenPackages = [];
-        $longestPackageKey = 0;
+        $availablePackages = $this->packageManager->getAvailablePackages();
+        ksort($availablePackages);
```

Comment:
i thought we decided to only show them in loading order?

```suggestion
```

Reply:
Yes 🤦🏽

Reply:
But then i would say we should change the loading order in the backend module too.

Reply:
Just to keep it consistent. Wdyt @mhsdesign